### PR TITLE
complete profile fix

### PR DESCRIPTION
**Pull request by shady-khokhar**

### CODE REVIEW

File: src/checkbox-group/CheckboxGroup.tsx [hunk 1]
```diff
@@ -79,31 +80,38 @@ const CheckboxGroup = ({
...
-            selected={selected}
-            setSelected={setSelected}
-            onChange={onChange}
-            sx={sx}
-            color={color}
-            {...option}
-          />
-        );
-      })}
+      {title ? (
```
Comment (debugger-anubhav) [inline src/checkbox-group/CheckboxGroup.tsx:90]: why is title optional?
Every checkbox section will always have some label text to be shown.

File: src/checkbox-group/styles.ts [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
 import { alpha, formControlLabelClasses } from "@mui/material";
 import Typography from "../typography";
+import Row from "src/row";
 
 export const Container = styled("div")({
   padding: 0,
   display: "flex",
   flexFlow: "column wrap",
-  margin: "12px 0",
+  // margin: "12px 0",
```
Comment (debugger-anubhav) [inline src/checkbox-group/styles.ts:10]: remove if not needed

File: src/checkbox-group/styles.ts [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
 });
+export const OptionsContainer = styled('div')<{direction?:'row'|'column'}>(({ theme, direction }) => ({
+  display: "flex",
+flexDirection:direction||'column',
```
Comment (debugger-anubhav) [inline src/checkbox-group/styles.ts:14]: formatting please

File: src/complete-profile/CompleteProfile.tsx [hunk 4]
```diff
@@ -752,16 +774,24 @@ const CompleteProfile = ({
                   </StyledLabel>
                   <StyledBox
                     sx={{
-                      paddingTop: "16px",
+                      // paddingTop: "16px",
```
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:777]: remove if not needed

File: src/complete-profile/CompleteProfile.tsx [hunk 5]
```diff
@@ -752,16 +774,24 @@ const CompleteProfile = ({
...
+                    <CheckboxGroup
+                      options={contactOptions}
+                      title={""}
+                      defaultChecked={[]}
+                      direction="row"
+                      onChange={(checked) =>
+                        onContactOptionChange && onContactOptionChange(checked)
+                      }
+                    />
+                    {/* <Checkbox label="Phone" color="black" />
```
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:793]: remove

File: src/complete-profile/CompleteProfile.tsx [hunk 6]
```diff
@@ -752,16 +774,24 @@ const CompleteProfile = ({
...
                       fontWeight: "500",
                     }}
                   >
-                    <Checkbox label="Email" color="black" />
-                    <Checkbox label="Phone" color="black" />
-                    <Checkbox label="Whatsapp" color="black" />
+                    <CheckboxGroup
+                      options={contactOptions}
+                      title={""}
+                      defaultChecked={[]}
```
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:787]: also it's okay if you don't pass defaultChecked. if nothing is checked by default, you can skip the prop

File: src/complete-profile/CompleteProfile.tsx (hunk 5 above)
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:793]: remove

File: src/complete-profile/CompleteProfile.tsx [hunk 7]
```diff
@@ -779,20 +809,29 @@ const CompleteProfile = ({
...
                   </StyledLabel>
                   <StyledBox sx={{ paddingTop: "16px" }}>
-                    <Checkbox label="$0-100" color="black" />
-                    <Checkbox label="$1000-2500" color="black" />
-                    <Checkbox label="$2500-5000" color="black" />
-                    <Checkbox label="$5000+" color="black" />
+                    <CheckboxGroup
+                      options={budgetOptions}
+                      title={""}
+                      defaultChecked={[]}
```
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:815]: likewise as above. you can skip it

File: src/complete-profile/CompleteProfile.tsx [hunk 8]
```diff
@@ -779,20 +809,29 @@ const CompleteProfile = ({
...
                     Estimated time for this project
                   </StyledLabel>
                   <StyledBox sx={{ paddingTop: "16px" }}>
-                    <Checkbox label="Less than 1 month" color="black" />
-                    <Checkbox label="1 -3 months" color="black" />
-                    <Checkbox label="More than 3 months" color="black" />
+                    <CheckboxGroup
+                      options={durationOption}
+                      title={""}
+                      defaultChecked={[]}
```
Comment (debugger-anubhav) [inline src/complete-profile/CompleteProfile.tsx:830]: likewise as above. you can skip it

File: src/complete-profile/CompleteProfile.tsx (hunk 7 above)
Comment (shady-khokhar) [reply src/complete-profile/CompleteProfile.tsx]: it was required fixed it as optional

File: src/checkbox-group/CheckboxGroup.tsx (hunk 1 above)
Comment (shady-khokhar) [reply src/checkbox-group/CheckboxGroup.tsx:90]: we have some specific styles required for the title which could be configured outside the checkbox

Comment (debugger-anubhav) [reply src/checkbox-group/CheckboxGroup.tsx:90]: props?

**Review by debugger-anubhav — COMMENTED** [review]
feedback 2

File: src/checkbox-group/CheckboxGroup.tsx (hunk 1 above)
Comment (debugger-anubhav) [reply src/checkbox-group/CheckboxGroup.tsx:90]: suggested change:

`{title && (
<Title
          color={placeholderColor}
          variant={titleVariant ?? "h6"}
          component="label"
        >
          {title}
        </Title>
)}`

**Review by debugger-anubhav — CHANGES_REQUESTED** [review]
iteration 3

**Review by debugger-anubhav — APPROVED** [review]

**Pull request merged**